README.md

```diff
@@ -1 +1,59 @@
-# Hello-world-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> This is a simple Hello Microverse project built with HTML & CSS
+
+
+## Built With
+
+- HTML
+- CSS
+
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+### Setup
+
+### Install
```

Comment:
- [ ] [OPTIONAL] Please kindly remove unused headers to make your project look neat.